src/resources/items.py

```diff
@@ -0,0 +1,22 @@
+from .base import DungeonBase
+from .character import Character
+
+
+class DungeonItem(DungeonBase):
+    """Dungeon Items that change players color if captured"""
+
+    used_items = []
+    unused_items = []
```

Comment:
If this is meant to be an abstract class for all items instead of a manager then remove these lists.

Reply:
Ill remove it for now but my idea was this was just for the color changing items. I was thinking of making a factory where we can simply pass in location, symbol and color. That way things dont get repeated